File: domain/src/main/java/com/aceplus/domain/model/forApi/sizeinstore/SizeInStoreShare.java
package com.aceplus.domain.model.forApi.sizeinstore;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by phonelin on 3/21/17.
 */

public class SizeInStoreShare {

    @SerializedName("SizeInStoreShareNo")
    @Expose
    private String sizeInStoreShareNo;
    @SerializedName("Date")
    @Expose
    private String date;
    @SerializedName("CustomerId")
    @Expose
    private Integer customerId;
    @SerializedName("StockId")
    @Expose
    private Integer stockId;
    @SerializedName("Quantity")
    @Expose
    private Integer quantity;
    @SerializedName("Status")
    @Expose
    private String status;
    @SerializedName("Remark")
    @Expose
    private String remark;
    @SerializedName("SaleManId")
    @Expose
    private Integer salemanId;

    private String customerNo;

    private String customerName;

    public String getSizeInStoreShareNo() {
        return sizeInStoreShareNo;
    }

    public void setSizeInStoreShareNo(String sizeInStoreShareNo) {
        this.sizeInStoreShareNo = sizeInStoreShareNo;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public Integer getCustomerId() {
        return customerId;
    }

    public void setCustomerId(Integer customerId) {
        this.customerId = customerId;
    }

    public Integer getStockId() {
        return stockId;
    }

    public void setStockId(Integer stockId) {
        this.stockId = stockId;
    }

    public Integer getQuantity() {
        return quantity;
    }

    public void setQuantity(Integer quantity) {
        this.quantity = quantity;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getRemark() {
        return remark;
    }

    public void setRemark(String remark) {
        this.remark = remark;
    }

    public Integer getSalemanId() {
        return salemanId;
    }

    public void setSalemanId(Integer salemanId) {
        this.salemanId = salemanId;
    }

    public String getCustomerNo() {
        return customerNo;
    }

    public void setCustomerNo(String customerNo) {
        this.customerNo = customerNo;
    }

    public String getCustomerName() {
        return customerName;
    }

    public void setCustomerName(String customerName) {
        this.customerName = customerName;
    }
}

File: domain/src/main/java/com/aceplus/domain/entity/sizeinstoreshare/SizeInStoreShare.kt
package com.aceplus.domain.entity.sizeinstoreshare

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


@Entity(tableName = "size_in_store_share")
class SizeInStoreShare {

    @PrimaryKey
    @ColumnInfo(name = "size_in_store_share_id")
    @SerializedName("size_in_store_share_id")
    @Expose
    var size_in_store_share_id: Int = 0

    @ColumnInfo(name = "customer_id")
    @SerializedName("customer_id")
    @Expose
    var customer_id: Int = 0

    @ColumnInfo(name = "invoice_date")
    @SerializedName("invoice_date")
    @Expose
    var invoice_date: String? = null

    @ColumnInfo(name = "stock_id")
    @SerializedName("stock_id")
    @Expose
    var stock_id: Int = 0

    @ColumnInfo(name = "quantity")
    @SerializedName("quantity")
    @Expose
    var quantity: Int = 0

    @ColumnInfo(name = "status")
    @SerializedName("status")
    @Expose
    var status: String? = null

    @ColumnInfo(name = "remark")
    @SerializedName("remark")
    @Expose
    var remark: String? = null

    @ColumnInfo(name = "sale_man_id")
    @SerializedName("sale_man_id")
    @Expose
    var sale_man_id: Int = 0

    @ColumnInfo(name = "delete_flag")
    @SerializedName("delete_flag")
    @Expose
    var delete_flag: String? = null

}